src/Entity/ResidentComplaintBoardCase.php

```diff
@@ -264,4 +264,16 @@ public function setFeePaid(bool $feePaid): self
 
         return $this;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFormattedAddress(string $type): ?string
+    {
+        if ('lease' === $type) {
+            return sprintf("%s\n%s %s", $this->getLeaseStreetNameAndNumber(), $this->getLeaseZip(), $this->getLeaseCity());
```

Comment:
If this is used for BBR API calls city should not be necessary - im not sure if it causes issues.

Reply:
For now it's only used for BBR API calls, but it makes general sense to be able to render a formatted address, e.g for future use in mail templates.